spec/api-crash-reporter-spec.js

```diff
@@ -180,12 +175,12 @@ describe('crashReporter module', () => {
             const crashUrl = url.format({
               protocol: 'file',
               pathname: path.join(fixtures, 'api', 'crash-restart.html'),
-              search: '?port=' + port
+              search: `?port=${port}`
             })
             w.loadURL(crashUrl)
           },
           processType: 'renderer',
-          done: done
+          done: done()
```

Comment:
js student asks: what does the addition of parenthesis do here?

Reply:
It calls `done()` function ), so the value of the `done` field becomes not the function itself, but its return value.
@codebytere are you sure you really wanted to do that?

Reply:
I made the change originally because thefunction (`done()`) weirdly wasn't getting called, so it would time out and fail consistently; do you know of a better solution? I'd rather keep it as was if possible, honestly

Reply:
If you call `done()` almost immediately without waiting for anything to happen in a test,
then, I'm afraid, your test is slightly less than completely useless )

I guess that the code that is supposes to call a callback you pass in the `done` field
doesn't do it.